### PR TITLE
Task 46 booking layout

### PR DESCRIPTION
**Pull request by alstdu**
Just humor me and double check over all of this. I spent like 2 hours on the form LOL my previous code was all over the place so I cleaned it up and looked up how to structure a form properly because the thing I found on W3Schools was NOT it. 

(ignore the fact that the text is on a new line)
image

### CODE REVIEW

**Comment by manhellis** [comment]
if conflict resovled then good

File: contact.html
Comment (manhellis) [inline contact.html:1]: i think this is the conflict, this shouldnt exist in my branch

**Review by manhellis — COMMENTED** [review]
remove change to contact page and merge should b ok !

**Comment by alstdu** [comment]
I fixed the conflict and added the social buttons

**Review by manhellis — APPROVED** [review]
nice

**Pull request merged**